test/randomdatagen.py

```diff
@@ -0,0 +1,53 @@
+# -*- coding: utf-8 -*-
+
+from collections import namedtuple
+from random import uniform, randint
+import sqlalchemy as sa
+import pandas as pd
+
+def genrandom():
```

Comment:
```suggestion
def generate_random_testing_data():
```

Good variables names describe what the variable.